Updates relating to 16.0.0-rc (#7038)
umbraco/UmbracoDocs: 16/umbraco-cms/reference/configuration/loggingsettings.md

By default, all log files are saved to the `umbraco/Logs` directory. You can define a custom directory for your log files by using the `Directory` key in the Logging settings.

Set the value to `~/LogFiles` to add all log files to a `LogFiles` directory in the root of the file structure.

## FileNameFormat

The default file name format for the Umbraco log file is `UmbracoTraceLog.{0}..json`. The single argument is replaced at runtime with the server's machine name.

> If you want to change the file name or include additional arguments, you can amend the format with this setting.

Reviewer: ```suggestion
If you want to change the file name or include additional arguments, you can amend the format with the `FileNameFormat` setting.
```

## FileNameFormatArguments

By default the single argument for the log file format name is the server's machine name.

> Other or additional arguments can be provided via this setting as a comma delimited string:

Reviewer: ```suggestion
Other or additional arguments can be provided via the `FileNameFormatArguments` setting using a comma-delimited string:
```
